controllers/api/deck-routes.js

```diff
@@ -226,18 +225,67 @@ router.delete('/:id', async (req, res) => {
   }
 });
 
+// Get a single card/array of cards by id (For getting the data to present to the front-ennd for the mathpix library).
+async function getCardData(req) {
+  try {
```

Comment:
This feels like duplication. The api end point for getting a deck brings all of it's cards along with it.

Reply:
Yes, I think that you are right. I guess that I do not entirely know what my line of reasoning at the time was. We should probably be fine to remove it!